src/Rector/MethodCall/ConvertRenderTemplateShortNotationToBundleSyntaxRector.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Symfony\Rector\MethodCall;
+
+use Nette\Utils\Strings;
+use PhpParser\Node;
+use PhpParser\Node\Arg;
+use PhpParser\Node\Expr\MethodCall;
+use PhpParser\Node\Scalar\String_;
+use PHPStan\Type\ObjectType;
+use PHPStan\Type\ThisType;
+use Rector\Core\Rector\AbstractRector;
+use Symplify\RuleDocGenerator\ValueObject\CodeSample\CodeSample;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+
+/**
+ * @see https://github.com/symfony/symfony/blob/4.4/src/Symfony/Bundle/FrameworkBundle/Templating/TemplateNameParser.php
+ * @see https://symfony.com/doc/4.4/templates.html#bundle-templates
+ *
+ * @see \Rector\Symfony\Tests\Rector\ClassMethod\TemplateAnnotationToThisRenderRector\TemplateAnnotationToThisRenderRectorTest
+ */
+
+final class ConvertRenderTemplateShortNotationToBundleSyntaxRector extends AbstractRector
```

Comment:
Might be nice to add this rule in symfony43.php as its fixing a deprecation there or should it be in symfony50.php as it is removed in there?